frontend/src/app/pages/home/home.component.ts

```diff
@@ -100,7 +102,9 @@ export class HomeComponent implements OnInit, OnDestroy {
     }
 
     try {
+      const token = await this._recaptchaV3Service.execute('patientRegistration').toPromise();
```

Comment:
to `patientRegistration` je odkud?

Reply:
to je name co se dava googlu a on mi to pak na BE vrati v requestu, nicmene protoze to pouzivame jenom na jednom miste, tak tohle nevaliduju